2-dom-manipulation/form.js

```diff
@@ -0,0 +1,80 @@
+const form = document.querySelector('form');
+
+//validate form data
+function validateForm(event) {
+    event.preventDefault();
+    const firstName = document.querySelector('input[id="firstName"]');
+    const lastName = document.querySelector('input[id="lastName"]');
+    const email = document.querySelector('input[id="email"]');
+    const telephone = document.querySelector('input[id="telephone"]');
+    const hobbies = document.querySelectorAll('input[type="checkbox"]');
```

Comment:
```suggestion
    const firstName = form.querySelector('input[id="firstName"]');
    const lastName = form.querySelector('input[id="lastName"]');
    const email = form.querySelector('input[id="email"]');
    const telephone = form.querySelector('input[id="telephone"]');
    const hobbies = form.querySelectorAll('input[type="checkbox"]');
```
Se puede usar la referencia del form para ir más rápido a los elementos del form.